public/app/features/browse-dashboards/components/RecentlyDeletedActions.tsx

```diff
@@ -29,20 +29,31 @@ export function RecentlyDeletedActions() {
       .map(([uid]) => uid);
   }, [selectedItemsState.dashboard]);
 
+  const dashboardOrigin: { [key: string]: string } = {};
```

Comment:
minor nit: might be slightly nicer to just type this as `Record<string, string>`

Reply:
🤔 I tried something like this before but must have made a tiny mistake because TS didn't stop complaining. Now it works. 👍